pkg/chrootarchive/archive_test.go

```diff
@@ -35,6 +36,13 @@ func CopyWithTar(src, dst string) error {
 	return chrootArchiver.CopyWithTar(src, dst)
 }
 
+func TestMain(m *testing.M) {
+	if reexec.Init() {
+		return
+	}
+	m.Run()
+}
```

Comment:
I would prefer that `TestMain` be moved to a new file `archive_freebsd_test.go` so that the reexec package isn't linked into the tests on other platforms.

Reply:
Sure, thanks!